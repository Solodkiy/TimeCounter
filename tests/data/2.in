                23:01 - 03:01
