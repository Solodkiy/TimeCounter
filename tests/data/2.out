23:01 - 03:01 = 4:00
                ----
                4:00